conans/client/graph/range_resolver.py

```diff
@@ -54,6 +54,7 @@ def _resolve_local(self, search_ref, version_range):
             # TODO: This is still necessary to filter user/channel, until search_recipes is fixed
             local_found = [ref for ref in local_found if ref.user == search_ref.user
                            and ref.channel == search_ref.channel]
+            local_found.extend(self._cache.editable_packages.edited_refs)
```

Comment:
Does edited_refs not need the filtering done in the line above this one?

Reply:
Yes, it does, I will extend the tests and fix this, very good catch!

Reply:
Fixed, please re-review